java/server/src/org/openqa/selenium/remote/server/commandhandler/UploadFile.java

```diff
@@ -60,8 +64,20 @@ public void execute(HttpRequest req, HttpResponse resp) throws IOException {
           "Expected there to be only 1 file. There were: " +
           (allFiles == null ? 0 : allFiles.length)));
     } else {
+      // IE requires multiple uploads to all be from the same directory - move files into one location
+      Path uploads_dir = Files.createDirectories(tempDir.toPath().getParent().resolve("remote_uploads"));
```

Comment:
Nit: Java naming conventions are CamelCase: `uploadsDir` instead of `uploads_dir`